src/igvfd/__init__.py

```diff
@@ -47,6 +47,8 @@ def static_resources(config):
         favicon_path = '/%s%s' % (config.route_prefix, favicon_path)
     config.add_route('favicon.ico', 'favicon.ico')
 
+    config.add_route('auditdoc', '/auditdoc')
```

Comment:
Can you create a /doc folder in the static folder in the package itself instead? https://github.com/IGVF-DACC/igvfd/tree/dev/src/igvfd/static Then https://{url}/static/doc/auditdoc should resolve it without having to create a route? For example right now this statically serves the favicon directly from folder I think: https://igvfd-dev.demo.igvf.org/static/img/favicon.ico